modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -10,6 +10,9 @@ import 'rxjs/add/operator/map';
 import * as models                                           from '../model/models';
 import { BASE_PATH, COLLECTION_FORMATS }                     from '../variables';
 import { Configuration }                                     from '../configuration';
+{{#withInterfaces}}
+import { I{{classname}} }                                    from './{{classname}}Interface';
```

Comment:
Giving an option to specify interface suffix and also the filename might be indeed useful for some projects. However in most cases, they will not care and just go with the default interface name whatever it is. Even here where we needed heavily custom usage of those interfaces, didn't need any specific naming.

This will also introduce unnecessary complexity for our code base for the 99% of use cases IMHO.